code/game/objects/effects/effect_system/foam.dm

```diff
@@ -99,7 +101,7 @@
 /datum/effect_system/foam_spread
 	var/amount = 5				// the size of the foam spread.
 	var/list/carried_reagents	// the IDs of reagents present when the foam was mixed
-	var/metal = 0				// 0=foam, 1=metalfoam, 2=ironfoam
+	var/metal = 0				// 0=foam, 1=metalfoam, 2=ironfoam, 3=razorburn
 
```

Comment:
defines pleaseeee

Reply:
Again, not smart enough to do, will work on later.

Reply:
done but I left the comments in and updated them